app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
```

Comment:
- [ ] The post belongs to an `author`, not `user`, and as such you kindly specify that. You can use the `class_name` attribute to refer to the User model.

  ```
  belongs_to :author, class_name: 'User'
  ```